vllm/v1/utils.py

```diff
@@ -147,8 +147,10 @@ def shutdown(proc: multiprocessing.Process, input_path: str, output_path: str):
 
 def bind_kv_cache(
     kv_caches: Dict[str, torch.Tensor],
-    forward_context: Dict[str, "Attention"],
     runner_kv_caches: List[torch.Tensor],
+    forward_context: Dict[str, "Attention"],
```

Comment:
change these parameters sequences can make more sense but in the other hand, it introduce more extra code changes.